templates/forms/premium-versions-upgrade-handler.php

```diff
@@ -20,11 +20,30 @@
     $plugin_name     = $plugin_data['Name'];
     $plugin_basename = $fs->get_plugin_basename();
 
+    $license = $fs->_get_license();
+
+    if ( ! is_object( $license ) ) {
+        $purchase_url = $fs->pricing_url();
+    } else {
+        $subscription = $fs->_get_subscription( $license->id );
+
+        $purchase_url = $fs->checkout_url(
+            is_object( $subscription ) ?
+                ( 1 == $subscription->billing_cycle ? WP_FS__PERIOD_MONTHLY : WP_FS__PERIOD_ANNUALLY ) :
+                WP_FS__PERIOD_LIFETIME,
+            false,
+            array( 'licenses' => $license->quota )
+        );
+    }
+
     $message = sprintf(
         fs_text_inline( 'There is a new version of %s available.', 'new-version-available-message', $slug ) .
-        fs_text_inline( ' %sRenew your license now%s to access version %s security & feature updates, and support.', 'renew-license-now', $slug ),
+        fs_text_inline( ' %s%s now%s to access version %s security & feature updates, and support.', 'renew-license-now', $slug ),
```

Comment:
@fajardoleo same comment as before regarding the translatable string (just include the `now` as part of the strings). Also, please notice that you've used `'buy-a-license'` for the key of the `''Buy a license'` string where previously it was `'buy-license'`. We need to be consistent across the SDK to use the same keys when the strings are identical.